### DISCUSSION TITLE
Support ggup format?

### DISCUSSION THREAD

**Discussion by hiqsociety** [mudler/LocalAI#1023] [Q&A]
llama.cpp uses ggup now.

can provide an eta on when this feature will be available?
thx in advance.

**Comment by mudler** [comment]
you mean gguf? gguf it's already supported

**Reply by y4my4my4m** [reply]
@mudler i havent' had any luck using GGUF. Are there special steps to take?
The backend should be `llama` and not `llama-stable` right? Apart from that, anything else? Can we use CUDA? Thread config, etc?

**Reply by mudler** [reply]
What's the error you have? gguf files should work and are tested automatically by the CI.
